### PR TITLE
Give method not found a primary span label

### PR DESCRIPTION
**Pull request by estebank**

### CODE REVIEW

File: src/librustc_typeck/check/method/suggest.rs [hunk 1]
```diff
@@ -425,6 +425,9 @@ impl<'a, 'tcx> FnCtxt<'a, 'tcx> {
                             "private field"
                         };
                         err.span_label(item_name.span, format!("{}, not a method", field_kind));
+                    } else if lev_candidate.is_none() && static_sources.is_empty() {
+                        err.span_label(span, format!("{} not found in `{}`", item_kind, ty_str));
+                        self.tcx.sess.trait_methods_not_found.borrow_mut().insert(orig_span);
                     }
                 } else {
                     err.span_label(span, format!("{} not found in `{}`", item_kind, ty_str));
```
Comment (Centril) [inline src/librustc_typeck/check/method/suggest.rs]: This is duplicated here; should be refactored to e.g. a method.

r=me with that addressed.

Comment (estebank) [reply src/librustc_typeck/check/method/suggest.rs]: Doing it but I don't see it making the code any cleaner :-/

Comment (Centril) [reply src/librustc_typeck/check/method/suggest.rs]: Hmm; I guess there's too much captured context relative the duplication in this case so it is probably indeed not worth it. Feel free to revert and r=me.

**Pull request merged**